utils/oscap.8

```diff
@@ -75,6 +75,11 @@ You may specify OVAL Definition files as the last parameter, XCCDF evaluation wi
 Select a particular profile from XCCDF document.
 .RE
 .TP
+\fB\-\-rule RULE\fR
+.RS
+Select a particular rule from XCCDF document (also requires profile to be specified). Only this rule will be evaluated.
+.RE
+.TP
```

Comment:
thank you for putting a note in the man page! This --rule concept has been asked for a lot, and documentation really helps get the word out.